.github/workflows/pr-to-update-winget.yml

```diff
@@ -0,0 +1,113 @@
+name: PR for updating winget
+
+# This action will run after a tag starting with "v" is published
+on:
+  push:
+    tags:
+      - "v*"
+  workflow_dispatch:
+
+jobs:
+  winget-update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout winget-automation
+        uses: actions/checkout@v4
+        with:
+          ref: winget-automation
+          token: ${{ secrets.KINVOLK_REPOS_TOKEN }}
+          # we need the full history for the git tag command, so fetch all the branches
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          user=${{github.actor}}
+          if [ -z $user ]; then
+            user=vyncent-t
+          fi
+          git config --global user.name "$user"
+          git config --global user.email "$user@users.noreply.github.com"
+
+      # Set up Node.js environment, pay attention to the version
+      # Some features might not be available in older versions
+      - name: Create node.js environment
+        uses: actions/setup-node@v4
+        with:
+          node-version: "21"
+
+      # Install the dependencies for the winget script
+      - name: Install winget dependencies
+        run: |
+          cd $GITHUB_WORKSPACE/app
+          npm install
+
+      # We set the latest tag as an environment variable before we use it in the next steps
+      # note that we have to echo the variable to the environment file to make it available in the next steps
+      - name: Set latest tag
+        run: |
+          echo "Setting latest tag"
+          latestTag=$(git tag --list --sort=version:refname 'v*' | tail -1)
+          # Remove the 'v' from the tag
+          latestTag=${latestTag#v}
+          echo "LATEST_HEADLAMP_TAG=$latestTag" >> $GITHUB_ENV
+          echo $latestTag
+
+      # checkout the winget-pkgs repository
+      - name: Checkout winget-pkgs
+        uses: actions/checkout@v4
+        with:
+          repository: headlamp-k8s/winget-pkgs
+          path: winget-pkgs
+          token: ${{ secrets.KINVOLK_REPOS_TOKEN }}
+          # we need the full history for the git tag command, so fetch all the branches
+          fetch-depth: 0
+
+      # Run the winget script
+      - name: Create winget package
+        run: |
+          echo "Running winget script"
+          echo "Repository: ${{ github.repository }}"
+          echo "Workspace: ${GITHUB_WORKSPACE}"
+          echo $GITHUB_WORKSPACE
+          pwd
+          echo "creating winget pkgs for ${LATEST_HEADLAMP_TAG}"
+          cd $GITHUB_WORKSPACE/app/windows/winget
+          node winget-create.js $LATEST_HEADLAMP_TAG $GITHUB_WORKSPACE/app/windows/winget
+          echo "Script finished"
+          ls
+          echo "Copying the winget package to the winget-pkgs repository"
+          cd $GITHUB_WORKSPACE/app/windows/winget/$LATEST_HEADLAMP_TAG
+          pwd
+          ls
+          cp -r $GITHUB_WORKSPACE/app/windows/winget/$LATEST_HEADLAMP_TAG $GITHUB_WORKSPACE/winget-pkgs/manifests/h/Headlamp/Headlamp
```

Comment:
I had mentioned in a previous comment that we should have the target fir the winbget-create.js be already the $GITHUB_WORKSPACE/winget-pkgs/manifests/h/Headlamp/Headlamp and avoid to create and then copy.